l4v-deploy/common.py

```diff
@@ -7,6 +7,7 @@
 
 import subprocess
 
+
```

Comment:
Accidental newline? The previous commit removes it.

Reply:
It got added by the style check -- I shouldn't have removed it in the first commit.

Reply:
Will see if I can reshuffle that.